index.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = (function() {
+	if (process.platform !== 'win32') {
+		return true;
+	}
+
+	var env = process.env;
+
+	if (env.APPVEYOR) {
```

Comment:
The only way to see log on AppVeyor, is open in browser. like: https://ci.appveyor.com/project/gucong3000/gulp-reporter/build/job/4dqfgqihdre4q87h

As you know, all browsers are supported unicode very well (even IE).